xbmc/video/VideoDatabase.cpp

```diff
@@ -2187,6 +2187,26 @@ int CVideoDatabase::UpdateDetailsForMovie(int idMovie, const CVideoInfoTag& deta
       }
     }
 
+    if (updatedDetails.find("showlink") != updatedDetails.end())
+    {
+      // remove existing links
+      vector<int> tvShowIds;
+      GetLinksToTvShow(idMovie, tvShowIds);
+      for (const auto& idTVShow : tvShowIds)
+        LinkMovieToTvshow(idMovie, idTVShow, true);
+
+      // setup links to shows if the linked shows are in the db
+      for (const auto& showLink : details.m_showLink)
+      {
+        CFileItemList items;
+        GetTvShowsByName(showLink, items);
+        if (!items.IsEmpty())
+          LinkMovieToTvshow(idMovie, items[0]->GetVideoInfoTag()->m_iDbId, false);
+        else
+          CLog::Log(LOGDEBUG, "VideoInfoScanner: Failed to link movie %s to show %s", details.m_strTitle.c_str(), showLink.c_str());
```

Comment:
First of all I'd make this a `LOGWARNING` because something went wrong.
Secondly you need to adjust the log message as this is not the VideoInfoScanner ;-)